README.md

```diff
@@ -51,3 +52,15 @@ To get acquainted with Codex, consider:
 ## API
 
 The client exposes a REST API that can be used to interact with the clients. Overview of the API can be found on [api.codex.storage](https://api.codex.storage).
+
+## Contributing and development
+
+Feel free to dive in, contributions are welcomed! Open an issue or submit PRs.
+
+### Linting and formatting
+
+`nim-codex` uses [nph](https://github.com/arnetheduck/nph) for formatting our code and it is requrired to adhere to its styling.
+If you are setting up fresh setup, in order to get `nph` run `make build-nph`.
+In order to format files run `make nph/<file/folder you want to format>`. 
+If you want you can install Git pre-commit hook using `make install-nph-commit`, which will format modified files prior commiting them. 
+If you are using VSCode and the NimLang extension you can enable "Format On Save" that will format the files using `nph`.
```

Comment:
Is this "Format On Save" or is it `nim.lintOnSave`? There is also `editor.formatOnSave` option, and perhaps it also has to be set to `true`, but `formatOnSave` is just a generic VSCode option if I remember correctly.

![image](https://github.com/user-attachments/assets/3e1eefa6-fed8-4d4c-9b0a-b0c3408d9a07)


Also, you could consider adding a link to the [NimLang](https://marketplace.visualstudio.com/items?itemName=NimLang.nimlang) to remove any doubts about the extension.

Reply:
![image](https://github.com/user-attachments/assets/0941424f-df59-4420-b9ce-91f26956f7fd)

Yeah it is, it is `nim.formatOnSave`. Will clarify though - thanks.

Reply:
Ok, I did some tests. So it looks like `nim.lintOnSave` is actually depreciated and `nim.formatOnSave` does exist on NimLang extension. Setting either `editor.formatOnSave` or `nim.formatOnSave` does the job (one of them need to be set to `true`, the order does not seem to matter).